### ISSUE TITLE
Use bigger chunks (RPC usage issue)

### ISSUE THREAD

**Issue by jennifermarie** [hyperlane-xyz/hyperlane-monorepo#1256]
From Discord:
@nambrot: "IIRC, we default to 1000 if not spcified in config. I believe 1000 was just done for Polygon nodes who have trouble to scan larger ranges. However, i just did the rollup ones and I'm pretty sure they were able to easily scan a range of 100k which obviously reduces the traffic and time needed to get to tip"

**Comment by mattiecnvr** [comment]
@nambrot Is there a place I should be looking at to find out what the max block range is for a given chain? I am guessing this is provider specific.

**Comment by tkporter** [comment]
Talked about this a bit with Mattie and we're of the opinion that this probably isn't worth doing

The benefit I see for this is for syncing with a wiped DB, which happens extremely rarely

Block range size rules eth_getLogs vary by provider / chain and can kinda get spooky-- providers tend to have different limits on the responses based off the block range size. E.g. [Alchemy](https://docs.alchemy.com/docs/deep-dive-into-eth_getlogs#making-a-request-to-eth_getlogs) says that if you do <= a 2k block range, you'll get at most 150mb, or if it's bigger than that you'll get only 10k logs. Infura has similar weird rules https://community.infura.io/t/eth-getlogs-block-range-too-large/3433 that encourage small block range sizes.

Basically:
* keeping the range small seems to be the more conservative option for not accidentally getting screwed by weird provider rules
* while a 2k block range does feel small, many providers use that as a cutoff for different types of behavior
* juggling different provider block range rules feels like a pita to avoid
* just generally doesn't feel like the best leverage of our time

**Comment by mattiecnvr** [comment]
Closing in favor of #1851
